cmd/installer/pack.go

```diff
@@ -531,7 +531,29 @@ func (p *PackType) resolveVersionModifier(pdscXML *xml.PdscXML) {
 			p.targetVersion = pdscXML.LatestVersion()
 			log.Debugf("- resolved (@^) as %s", p.targetVersion)
 		} else {
-			log.Errorf("Tried to install major version %s.x.x, highest available major version is %s.x.x", p.Version[:1], pdscXML.LatestVersion()[:1])
+			log.Errorf("No compatible minor version available for pack version >= %s, highest available major version is %s.x.x", p.Version, utils.SemverMajor(pdscXML.LatestVersion()))
+		}
+		return
+	}
+
+	// The next tricky one is @~, because it needs to be the latest
```

Comment:
Is this a real misunderstanding?
It must be the latest only if there is no other pack installed already which satisfies the >= condition.